File: ip-messaging/rest/services/create-service/create-service.5.x.rb
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

client = Twilio::REST::Client.new(account_sid, auth_token)

# Create a new service and make note of that service's SID.
new_service = client.chat.v2
                    .services.create(friendly_name: 'An example service')
puts "Created Chat service #{new_service.sid}"

File: proxy/create-service/create-service.5.x.cs
using System;
using Twilio;
using Twilio.Rest.Preview.Proxy.Service.Session;

static void Main(string[] args)
{
    // Find your Account Sid, Auth Token and Proxy Service sid at twilio.com/console
    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    const string authToken = "your_auth_token";
    const string proxyServiceSid = "KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    const string sessionSid = "KCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    TwilioClient.Init(accountSid, authToken);

    var proxyService = ServiceResource.Create("FRIENDLY_NAME");

    Console.WriteLine(proxyService.Sid);
}
